merging-configmaps-gitops/README.md

```diff
@@ -0,0 +1,75 @@
+# Merging Configmaps in a gitops context
```

Comment:
I believe this applies to both Secrets and ConfigMaps so maybe it would be more accurate to just call it "config" or "app values".

Reply:
True - I didn't consider this but actually a really good point.

Reply:
Added a disclaimer :)